.gitignore

```diff
@@ -7,3 +7,4 @@
 # IntelliJ
 /.idea
 *.iml
+.toDelete
```

Comment:
what is it?

Reply:
some useless file generated by intellij I don't know why but it's annoying because it's tracked

Reply:
ok